tests/logitech_receiver/test_hidpp20_complex.py

```diff
@@ -110,6 +110,49 @@ def test_FeaturesArray_get_feature(device, expected0, expected1, expected2, expe
     assert result5v == expected5v
 
 
+@pytest.mark.parametrize(
+    "device, expected_result",
+    [
+        (device_zerofeatures, []),
+        (
+            device_standard,
+            [
+                (hidpp20_constants.FEATURE.ROOT, 0),
+                (hidpp20_constants.FEATURE.FEATURE_SET, 1),
+                (common.NamedInt(256, "unknown:0100"), 2),
+                (None, 3),
+                (None, 4),
+                (hidpp20_constants.FEATURE.REPROG_CONTROLS_V4, 5),
+            ],
+        ),
+    ],
+)
+def test_FeaturesArray_enumerate(device, expected_result):
```

Comment:
Nit: lower case names

Reply:
This name is taken from the name of the class and should match exactly.

Reply:
No it should never be a mix and match. That‘s the easiest to stay consistent and stick with

Reply:
Such name clashes are even good, because it shows you, that there's already a test. There shouldn't exist function names, global variables, that have the same name and could be mistakenly used.

As there's no combination of camelCase and snake_case, that is commonly used, and it's ulgy too. I will never use such a spelling and stick to snake_case for all pytest test functions.